src/nrepl/index.ts

```diff
@@ -372,8 +372,12 @@ export class NReplSession {
     });
   }
 
-  async switchNS(ns: any) {
-    await this.eval(`(in-ns '${ns})`, this.client.ns).value;
+  async evaluateInNs(nsForm: string, ns: string) {
+    try {
+      await this.eval(nsForm, ns).value;
```

Comment:
Why is the first arg called `nsForm`? This, generically, could be any code form, right?

Reply:
Good catch. I renamed and changed this function quite a bit while working on this. Maybe the best is to rename the function to indicate that it is only really meant to evaluate the ns form. For other cases, we have the eval function already.